src/manual-edits/content/docs/new-relic-only/tech-writer-style-guide/article-templates/landing_page_template.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Landing page template'
+tags:
+translate:
+redirects:
```

Comment:
By the way, these fields that you have blank are optional. `title` is the only required field on this bit of content. You can omit the rest of these if you don't have values for them to keep it clean.

Reply:
Jerel, I accepted your suggestion.